src/cript/api/api.py

```diff
@@ -402,6 +406,791 @@ def api_prefix(self):
     def api_version(self):
         return self._api_version
 
+    # ==========================================================
+
+    @staticmethod
+    def remove_keys_from_dict(
+        dictionary,
+        keys_to_remove=[
+            "admin",
+            # "uuid",
+            "updated_by",
+            "created_by",
+            "member",
+            "experiment_count",
+            "inventory_count",
+            "public",
+            "created_at",
+            "email",
+            "model_version",
+            "orcid",
+            "uid",
+            "updated_at",
+            "username",
+            "admin_count",
+            "collection_count",
+            "locked",
+            "material_count",
+            "member_count",
+            "component_count",
+            "computational_forcefield_count",
+            "identifier_count",
+            "computation_count",
+            "component_count",
+            "citation_count",
+            "condition_count",
+            "sample_preparation_count",
+            "property_count",
+        ],
+    ):
+        """Recursively remove keys from a dictionary."""
+        if not isinstance(dictionary, dict):
+            return dictionary
+        for key in keys_to_remove:
+            dictionary.pop(key, None)
+        for key, value in list(dictionary.items()):  # Use list() to avoid RuntimeError during iteration
+            if isinstance(value, dict):
+                API.remove_keys_from_dict(value, keys_to_remove)
+            elif isinstance(value, list):
+                for item in value:
+                    API.remove_keys_from_dict(item, keys_to_remove)
+        return dictionary
+
+    @classmethod
+    def add_to_dict(cls, dictionary, key=None, value=None):
```

Comment:
can we put this in `util`

It has no relation to the class, other then that you use it there.